app/models/miq_widget_set.rb

```diff
@@ -5,6 +5,8 @@ class MiqWidgetSet < ApplicationRecord
 
   WIDGET_DIR =  File.expand_path(File.join(Rails.root, "product/dashboard/dashboards"))
 
+  scope :default, -> { find_by(:name => 'default', :read_only => true) }
```

Comment:
Should not be here `where(:name => 'default', :read_only => true)`  when this is a scope ?

Reply:
Well, there's just one single dashboard that's default...so I don't think so.

Reply:
so then you probably want to define method,  something like `find_default_dashboard`, scopes are (by their very nature) chainable.